composer.json

```diff
@@ -56,7 +56,7 @@
     },
     "require-dev": {
         "swoole/ide-helper": "4.5.5",
-        "appwrite/sdk-generator": "0.2.3",
+        "appwrite/sdk-generator": "dev-master",
```

Comment:
You want this for production?

Reply:
Ohh no! This was just me testing things out :)